galaxy/importer/tests/test_linters.py

```diff
@@ -69,6 +74,42 @@ def test_flake8_fail():
         ]
         expected = [s.format(fp.name) for s in expected]
     assert sorted(result) == sorted(expected)
+    assert 'W504' in error_ids
+    assert 'E117' in error_ids
+    assert 'F821' in error_ids
+    assert 'undefined name' in ' '.join(descriptions).lower()
+    assert 'over-indented' in ' '.join(descriptions).lower()
+
+
+def test_flake8_error_id_match():
+    linter = linters.Flake8Linter()
+    msg_e501 = 'file.py:99:80: E501 line too long (81 > 79 characters)'
+    msg_f401 = 'forms.py:5:1: F401 "pyaml" imported but unused'
+    msg_w191 = 'plugin.py:7:2: W191 indentation contains tabs'
+
+    error_id, _ = linter.parse_id_and_desc(msg_e501)
+    assert error_id == 'E501'
```

Comment:
You're testing `parse_id_and_desc` function here. Maybe it makes sense to check if message is parsed correctly as well?